conf/mts.toml

```diff
@@ -0,0 +1,72 @@
+
+# Default broker URL connecting to a local RabbitMQ instance for testing.
+# Set this to a specific vhost in deployed instance.
+amqp_url = "amqp://"
+
+# All these three configs are for receiving messages from fedora-messaging.
+# Set them to specific certificate and key file in deployed instance.
+# [tls]
+# ca_cert = "/etc/fedora-messaging/cacert.pem"
+# keyfile = "/etc/fedora-messaging/fedora-key.pem"
+# certfile = "/etc/fedora-messaging/fedora-cert.pem"
+
+[client_properties]
+app = "Message Tagging Service"
+# Some suggested extra fields:
+# URL of the project that provides this consumer
+app_url = "https://github.com/fedora-modularity/message-tagging-service"
+# Contact emails for the maintainer(s) of the consumer - in case the
+# broker admin needs to contact them, for e.g.
+app_contacts_email = ["cqi@redhat.com"]
+
+[exchanges."amq.topic"]
+type = "topic"
+durable = true
+auto_delete = false
+arguments = {}
+
+[queues.8bbfbdd1-02c1-4d18-a0ee-1752da519a17]
+durable = false
+auto_delete = true
+exclusive = true
+arguments = {}
+
+[[bindings]]
+queue = "8bbfbdd1-02c1-4d18-a0ee-1752da519a17"
```

Comment:
Is there a way to not have to repeat the this value?

Reply:
Seems no way to avoid repeat in config file.

https://fedora-messaging.readthedocs.io/en/latest/quick-start.html